src/Stratis.Bitcoin/Connection/PeerBanning.cs

```diff
@@ -23,6 +23,14 @@ public interface IPeerBanning
         /// <param name="reason">An optional reason for the ban, the 'reason' is only use for tracing.</param>
         void BanAndDisconnectPeer(IPEndPoint endpoint, int banTimeSeconds, string reason = null);
 
+        /// <summary>
+        /// Overload for the BanAndDisconnectPeer method that uses the connection manager's default ban interval.
```

Comment:
The method has different name, so it doesn't look like overload. And when it is overload, you don't need to have it in a comment,

Reply:
Agree I would just keep the name as is.